Update

hiramRV opened the pull request:
- Cambio de orden datos 2020-05-09 y 2020-05-08.
- Número de recuperados para el 2020-05-10
- Corrimiento de recuperados a partir de 2020-05-10
- Número de casos recuperados correcto para el 2020-06-16

Fuente: https://www.mspas.gob.gt/index.php/noticias/covid-19/casos 

[infocovid19.xlsx](https://github.com/ncovgt2020/ncovgt2020/files/4813745/infocovid19.xlsx)

Dejo un archivo de ejemplo (El de los casos hasta el 21/06/2020).

hiramRV committed:
Update

- Cambio de orden datos 2020-05-09 y 2020-05-08.
- Número de recuperados para el 2020-05-10
- Corrimiento de recuperados a partir de 2020-05-10
- Número de casos recuperados correcto para el 2020-06-16

ncovgt2020 commented:
@hiram22  Porfavor revisa #2 y actualiza el pull request luego de los cambios solicitados ahí. ¡Gracias!

hiramRV committed:
Update del Pull Request

- Se corrige el orden de dos fechas
- Se corrige la cantidad de recuperados (914)

(Fuente https://www.mspas.gob.gt/index.php/noticias/covid-19/casos )
